src/v3/src/transformer/phone/phoneChallengeTransformer.test.ts

```diff
@@ -57,7 +57,7 @@ describe('PhoneChallengeTransformer Tests', () => {
       .toBe('oie.phone.verify.title');
     expect(updatedFormBag.uischema.elements[2].type).toBe('Description');
     expect((updatedFormBag.uischema.elements[2] as DescriptionElement).options.content)
-      .toBe(`oie.phone.verify.sms.codeSentText <span class="strong no-translate">${redactedPhone}.</span> oie.phone.verify.enterCodeText`);
+      .toBe(`oie.phone.verify.sms.codeSentText <span class="strong no-translate">&lrm;${redactedPhone}.</span> oie.phone.verify.enterCodeText`);
```

Comment:
using the `&lrm;` unicode mark also keeps the period in the correct spot: https://www.w3.org/TR/WCAG20-TECHS/H34.html